base/start-notebook.d/ope-sartup.sh

```diff
@@ -68,7 +68,16 @@ if [[ -d $MOUNT_DIR ]]; then
   	  echo "$SN: Found $MOUNT_DIR/.myjupyter_start.sh: sourcing it"
 	  . $MOUNT_DIR/.myjupyter_start.sh
       fi
-   fi   
+
+	  OPE=ope
```

Comment:
we can create a work directory `ope_work` inside `ope` directory for users to create their content in persistent storage. Then we link `/home/ope/ope_work/` to `/opt/app-root/src/ope_work`:
```
      OPE=ope_work
      if [[ ! -d home/ope/$OPE ]]; then
	  if [[ ! -d $MOUNT_DIR/$OPE ]]; then
              echo "$SN: creating $MOUNT_DIR/$OPE"
	      if mkdir $MOUNT_DIR/$OPE; then
	          echo "$SN: Linking $MOUNT_DIR/$OPE -> /home/ope/$OPE"
    	          ln -s $MOUNT_DIR/$OPE /home/ope/$OPE
	      fi
          fi
      fi
```